Source/Core/Core/DSP/Jit/DSPEmitter.cpp

```diff
@@ -333,7 +333,7 @@ void DSPEmitter::Compile(u16 start_addr)
   {
     m_block_links[start_addr] = m_block_link_entry;
 
-    for (u16 i = 0x0000; i < 0xffff; ++i)
+    for (size_t i = 0; i < 0xffff; ++i)
```

Comment:
Yeah, in my original code, before I did self-review, I had used `.size()`, but I didn't know if it was specifically done this way on purpose, or was just a typo (`<` instead of `<=`), so I erred on the side of caution here.